man/systemd.netdev.xml

```diff
@@ -983,6 +986,100 @@
     as the <literal>[Tun]</literal> section.</para>
   </refsect1>
 
+  <refsect1>
+    <title>[WireGuard] Section Options</title>
+
+    <para>The <literal>[WireGuard]</literal> section accepts the following
+    key:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>PrivateKey=</varname></term>
+        <listitem>
+          <para>The private key for the interface. It can be generated
```

Comment:
is this encoded in base64? if so, please say so, after all you say so for the PublicKey= setting